### PR TITLE
Minimize bundle size for optimization. Updated sh script

### PR DESCRIPTION
**Pull request by aleksebastian**
Added babel transform runtime plugin. Used chance instead of faker to minimize bundle size. Changed deploy command to zip file before deploying.

### CODE REVIEW

File: checkout-service/public/index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
   </head>
   <body>
     <div id='checkout'></div>
-    <script type='text/javascript' src='checkout_bundle.js'></script>
+    <script type='text/javascript' src='https://fec-blitva-checkout.s3.us-east-2.amazonaws.com/checkout_bundle.js.gz'></script>
```
Comment (jrozal) [inline checkout-service/public/index.html:9]: Just curious about your decision to use the compressed bundle file in the script tag as opposed to using [express to serve file](https://stackoverflow.com/questions/55748479/how-to-use-highly-compressed-bundle-file-bundle-tar-gz-for-hosting-in-server)? I didn't find much out there regarding best practice, but most I found suggested to use express (possibly for cross-browser compatibility?). I'm not certain and was curious about your thoughts as I am using gz compressed files for optimization as well!

File: checkout-service/client/src/components/SecureTransactionAndSellerDetails.jsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import React, { useState } from 'react';
 import styled from 'styled-components';
-import faker from 'faker';
+import Chance from 'chance';
+const chance = new Chance();
```
Comment (jrozal) [inline checkout-service/client/src/components/SecureTransactionAndSellerDetails.jsx:4]: refactor looks good 👍

File: checkout-service/client/src/components/AddToListDropDown.jsx [hunk 3]
```diff
@@ -112,7 +113,7 @@ const generateListItems = () => {
   let listItems = [];
   for (let i = 0; i < randomNumOfLists; i++) {
     let randomPrivateOrPublicIndex = Math.floor(Math.random() * Math.floor(2));
-    listItems.push(<ListItem><ListItemImgWrapper><ListItemImg src={faker.image.imageUrl()}></ListItemImg></ListItemImgWrapper><ListItemTextWrapper><ListItemLabel>{faker.random.word()}</ListItemLabel><ListItemStatus>{listStatus[randomPrivateOrPublicIndex]}</ListItemStatus></ListItemTextWrapper></ListItem>);
+    listItems.push(<ListItem><ListItemImgWrapper><ListItemImg src={'https://placeimg.com/50/50'}></ListItemImg></ListItemImgWrapper><ListItemTextWrapper><ListItemLabel>{chance.word()}</ListItemLabel><ListItemStatus>{listStatus[randomPrivateOrPublicIndex]}</ListItemStatus></ListItemTextWrapper></ListItem>);
   };
   return listItems;
 };
```
Comment (jrozal) [inline checkout-service/client/src/components/AddToListDropDown.jsx:119]: Refactored code using chancejs looks good 👍

**Review by jrozal — COMMENTED** [review]
Hi Alek, refactored code using chancejs looks good, especially since it helps minimize your bundle size. You may want to consider also adding the compressed gz bundle file to your gitignore as well.

File: checkout-service/public/index.html (hunk 1 above)
Comment (aleksebastian) [reply checkout-service/public/index.html:9]: Hey Justin! So I think server side rendering with express might be best. For this project however, I wanted for the bundle to be available at all times without having to keep my instance up. Serving the bundle with S3 allows me to do so.

Comment (jrozal) [reply checkout-service/public/index.html:9]: Makes perfect sense, thank you for sharing!

**Pull request merged**